Library/Formula/cake.rb

```diff
@@ -0,0 +1,47 @@
+class Cake < Formula
+  desc "Cake (C# Make) is a build automation system with a C# DSL."
+  homepage "http://cakebuild.net/"
+  url "https://www.nuget.org/api/v2/package/Cake/#{version}"
```

Comment:
We don't use `#{version}"` in URLs. Should just be `https://www.nuget.org/api/v2/package/Cake/0.5.1`, Thanks.